CMakeLists.txt

```diff
@@ -1888,9 +1888,8 @@ add_custom_command(
   ${post_build_commands}
   BYPRODUCTS
   ${post_build_byproducts}
-  COMMENT "Generating files from ${KERNEL_ELF_NAME} for board: ${BOARD}"
+  COMMAND ${CMAKE_COMMAND} -E echo "Generating files from ${PROJECT_BINARY_DIR}/${KERNEL_ELF_NAME} for board: ${BOARD}"
```

Comment:
Somehow the memory report is still printed first:
![image](https://github.com/zephyrproject-rtos/zephyr/assets/89006246/130d6509-3670-436c-9db0-4c986f867603)

Reply:
> Somehow the memory report is still printed first:

Because the memory report is printed by the linker when using gcc, and is not a postbuild command.
It will only help in relation to other postbuild commands, such as `image.py` 😉